### DISCUSSION TITLE
Add dedicated trigger for timers

### DISCUSSION THREAD

**Discussion by chelming** [home-assistant/frontend#20037] [Automations / scripts / scenes]
Right now the method for triggering an automation based on a timer isn't very user-friendly. It requires you to choose the manual event trigger, know the name of the event you want to trigger off of (e.g., `timer.finished`), and then know that you have to format the event data as `entity_id: timer.your_timer`.

It would be nice if there was a trigger for timer where you pick the entity id of the timer using an entity id picker, then choose the timer event that you want to trigger off of.

Sample of what it might look like:
![image](https://github.com/home-assistant/frontend/assets/7746625/5d9e6751-93a4-4377-982f-18e4d42c68e8)

**Comment by karwosts** [comment]
Related: https://github.com/home-assistant/frontend/pull/19252

**Reply by chelming** [reply]
oh, sick! I searched discussions but not PRs 😆 

nice work, @karwosts ❤️
